Makefile

```diff
@@ -172,6 +172,7 @@ $(FMT):
 
 fmt: TO_FMT ?= $(shell find applications core -iname '*.erl' -or -iname '*.hrl' -or -iname '*.app.src')
 fmt: $(FMT)
+	@ERL_LIBS=deps/:core/:applications/ $(ROOT)/scripts/no_raw_json.escript
```

Comment:
I'd say this should go into a `code_checks` target which you should create.
`no_raw_json` doesn't write to files, does it?